runtime/src/bank/tests.rs

```diff
@@ -7070,15 +7070,19 @@ fn test_reconfigure_token2_native_mint() {
 fn test_bank_load_program() {
     solana_logger::setup();
 
-    let (genesis_config, _) = create_genesis_config(1);
+    let (genesis_config, mint_keypair) = create_genesis_config_no_tx_fee(1_000_000_000);
     let bank = Bank::new_for_tests(&genesis_config);
+    let (bank, bank_forks) = bank.wrap_with_bank_forks_for_tests();
+    goto_end_of_slot(bank.clone());
+    let bank = new_bank_from_parent_with_bank_forks(&bank_forks, bank, &Pubkey::default(), 42);
+    let bank = new_bank_from_parent_with_bank_forks(&bank_forks, bank, &Pubkey::default(), 50);
```

Comment:
Do you need two of these here? Can't you just warp to 50?

Reply:
We have to insert 42 into the fork graph so that it is visible from 50.